core/src/main/java/com/linecorp/armeria/common/logging/JsonLogFormatter.java

```diff
@@ -131,62 +135,68 @@ public String formatRequest(RequestOnlyLog log) {
             }
 
             final ObjectNode objectNode = objectMapper.createObjectNode();
-            objectNode.put("startTime",
+            if (containContext) {
+                objectNode.put("ctx", ctx.toString());
+            }
+            final ObjectNode requestNode = objectMapper.createObjectNode();
+            objectNode.set("request", requestNode);
```

Comment:
What do you think of setting `request` as a property and move other properties under the root node?

Reply:
For example:
```
{
  "type": "request",
  "startTime": ...,
  "length": ...,
  ...
}
```